CodeStarterCamp_Week1/Step3.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Step3.swift
+//  CodeStarterCamp_Week1
+//
+//  Created by Hisop on 2023/07/14.
+//
+
+import Foundation
+
+func printOutInfo(height: Int, body: String, topping: String = " ", barHeight: Int) {
+    print("길이: \(height)")
+    print("몸통: \(body)")
+    print("토핑: \(topping)")
+    print("막대길이: \(barHeight)")
+    print("")
```

Comment:
Hisop 님은 멀티라인에 대해서 알고 계시나요?

```suggestion
    let info: String = """
    길이: \(height)
    몸통: \(body)
    토핑: \(topping)
    막대길이: \(barHeight)
    
    """
    
    print(info)
```

이런식으로도 사용이 가능하니 참고해주세요 😄